README.md

```diff
@@ -31,16 +31,17 @@ Environment variables
 
 The following environment variables are exposed for InfraMock:
 
-`POPULATE_MOCK`: whether localstack should be filled with a mocked PBMC data set. If this is
-set to `true`, InfraMock will try to use a locally running version of `api` to populate the
-localstack DynamoDB database with using its `/experiments/generate` endpoint. For this to work,
-`api` **must** be running with `CLUSTER_ENV` set to `development`, which is the default behavior. The default value is `true`.
+`POPULATE_MOCK`: whether localstack should be filled with a mocked PBMC data set. This is
+set to `true` by default, which means that InfraMock will try to use a locally running version of 
+the `api` to populate the localstack DynamoDB database with using its `/experiments/generate` endpoint. 
+For this to work, `api` **must** be running with `CLUSTER_ENV` set to `development`, which is the default behavior.
 
 `MOCK_EXPERIMENT_DATA_PATH`: where to get the mocked data for upload to the local S3 from. If
-this is not set, it will default to the GitLab URL where the data set used for the `worker` unit
+this is not set, it will default to the Github URL where the dataset used for the `worker` unit
 test is located.
 
-`AWS_DEFAULT_REGION`: the default mocked region for your infrastructure to be deployed under. If it's not set, it defaults to `eu-west-1`.
+`AWS_DEFAULT_REGION`: the default mocked region for your infrastructure to be deployed under. If it's not set, 
+it defaults to `eu-west-1`.
 
 FAQ
```

Comment:
@marcellp for the answer of question 2 under the FAQ, do we run `aws` while running inframock on a separate terminal?

Reply:
Correct, while inframock as a service is running, you can just run `aws` locally wherever (in VS code, another terminal, etc.) and it will connect to it.

Reply:
ok cool, I will update the docs to clarify this